### PR TITLE
migration, seeder & eloquent relationship

### PR DESCRIPTION
**Pull request by phuongsai**
Setup database:
1. Create migration file for Tasks table.
2. Seeding database: Model Task & User.
3. Eloquent relationship: Task & User (One to Many).

### CODE REVIEW

**Comment by phuongsai** [comment]
ready

File: app/Task.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Task extends Model
+{
+    use SoftDeletes;
+
+    protected $dates = ['deleted_at'];
+
+    // mass assignable value
+    protected $fillable = ['name', 'description', 'user_id'];
```
Comment (quoctd-0607) [inline app/Task.php:15]: protected $fillable = [
'name', 
'description', 
'user_id',
];

Comment (phuongsai) [reply app/Task.php]: Fixed!

**Comment by phuongsai** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**